### PR TITLE
BAEL-5634: move the code to a different module

### PR DESCRIPTION
**Pull request by azhwani**
Move https://www.baeldung.com/java-hashmap-inside-list code to core-java-collections-list-4

### CODE REVIEW

File: core-java-modules/core-java-collections-list-4/src/test/java/com/baeldung/list/listOfHashMaps/ListOfHashMapsUnitTest.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        phpBooksAuthorsMap.put("Head First PHP & MySQL", Arrays.asList("Lynn Beighley", "Michael Morrison"));
+
+        booksAuthorsMapsList.add(javaBooksAuthorsMap);
+        booksAuthorsMapsList.add(phpBooksAuthorsMap);
+
+        assertTrue(booksAuthorsMapsList.get(0)
+            .keySet()
+            .containsAll(javaBooksAuthorsMap.keySet()
+                .stream()
+                .collect(Collectors.toList())));
```
Comment (JavierPeris) [inline core-java-modules/core-java-collections-list-4/src/test/java/com/baeldung/list/listOfHashMaps/ListOfHashMapsUnitTest.java:34]: You can simplify this code using ArrayList constructor so: 
```
        assertTrue(booksAuthorsMapsList.get(0)
            .keySet()
            .containsAll(new ArrayList<>(javaBooksAuthorsMap.keySet())));
```

File: core-java-modules/core-java-collections-list-4/src/test/java/com/baeldung/list/listOfHashMaps/ListOfHashMapsUnitTest.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+        assertTrue(booksAuthorsMapsList.get(1)
+            .keySet()
+            .containsAll(phpBooksAuthorsMap.keySet()
+                .stream()
+                .collect(Collectors.toList())));
```
Comment (JavierPeris) [inline core-java-modules/core-java-collections-list-4/src/test/java/com/baeldung/list/listOfHashMaps/ListOfHashMapsUnitTest.java:40]: Same here

File: core-java-modules/core-java-collections-list-4/src/test/java/com/baeldung/list/listOfHashMaps/ListOfHashMapsUnitTest.java (hunk 1 above)
Comment (azhwani) [reply core-java-modules/core-java-collections-list-4/src/test/java/com/baeldung/list/listOfHashMaps/ListOfHashMapsUnitTest.java]: Done

File: core-java-modules/core-java-collections-list-4/src/test/java/com/baeldung/list/listOfHashMaps/ListOfHashMapsUnitTest.java (hunk 2 above)
Comment (azhwani) [reply core-java-modules/core-java-collections-list-4/src/test/java/com/baeldung/list/listOfHashMaps/ListOfHashMapsUnitTest.java]: Done

File: core-java-modules/core-java-collections-list-4/src/test/java/com/baeldung/list/listOfHashMaps/ListOfHashMapsUnitTest.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
+package com.baeldung.list.listOfHashMaps;
```
Comment (JavierPeris) [inline core-java-modules/core-java-collections-list-4/src/test/java/com/baeldung/list/listOfHashMaps/ListOfHashMapsUnitTest.java:1]: Hi @azhwani , we know that we could use camel case here but our convention is lower case. So could you rename the package to all lower case?

**Review by JavierPeris — APPROVED** [review]

**Pull request merged**